src/js/components/modals/InstallPackageModal.js

```diff
@@ -293,14 +305,60 @@ class InstallPackageModal
     );
   }
 
+  getPreinstallNotes(notes, truncated) {
+    if (truncated && notes) {
```

Comment:
wanna return `null` if `!notes`? then we won't have this basically empty `p` tag